modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-disk-partitioning_{context}"]
+= Configuring disk partitioning with SiteConfig
+
+You can use the `SiteConfig` YAML file to generate the `MachineConfig` CR used for disk partitioning. Prior to installation, you need to modify values in the `SiteConfig` CR to reflect dependencies on the underlying disk.
+
+[NOTE]
+====
+You must use persistent naming for devices to avoid device names such as `/dev/sda` and `/dev/sdb` being switched at every reboot. You can use `rootDeviceHints` to choose the bootable device and then use same device for further partitioning: in this case, for Image registry.
+====
+
+.Prerequisites
+
+
+* You have installed and configured Zero Touch Provisioning (ZTP). For information about this, see the topic on ZTP in Additional resources.
+
+.Procedure
+
+. Add the following YAML to the `SiteConfig.yaml` file to generate `MachineConfig` for disk partitioning. :
++
+[source,yaml]
```

Comment:
Suggest reviewing the updated notes in https://github.com/openshift/openshift-docs/pull/53083/files#diff-c0c2580378cc5e557211fa281f26dbe3efdd1da52e33bc7ef3873f9f10639307 and making appropriate changes here.